lib/rules/no-invalid-role.js

```diff
@@ -141,28 +141,47 @@ const COMPOSITE_WIDGET_ROLES = new Set([
 const DOCUMENT_STRUCTURE_ROLES = new Set([
```

Comment:
Can you add a comment above this array with the URL this list is supposed to come from?

Reply:
Good idea, added!